src/components/AnnotationPopup/AnnotationPopup.js

```diff
@@ -337,37 +339,39 @@ const AnnotationPopup = () => {
         ) : (
           <DatePicker onClick={handleDateChange} annotation={firstAnnotation} onDatePickerShow={onDatePickerShow} />
         )
-      ) : (shortCutKeysFor3DVisible && firstAnnotation instanceof Annotations.Model3DAnnotation) ?
-        <ShortCutKeysFor3DComponent /> : (
-          <CustomizablePopup dataElement="annotationPopup">
-            {showCommentButton && (
-              <ActionButton
-                dataElement="annotationCommentButton"
-                title="action.comment"
-                img="icon-header-chat-line"
-                onClick={commentOnAnnotation}
-              />
-            )}
-            {showEditStyleButton && (
-              <ActionButton
-                dataElement="annotationStyleEditButton"
-                title="action.style"
-                img="icon-menu-style-line"
-                onClick={() => setIsStylePopupOpen(true)}
-              />
-            )}
-            {firstAnnotation.ToolName === 'CropPage' && (
-              <ActionButton
-                dataElement="annotationCropButton"
-                title="action.apply"
-                img="ic_check_black_24px"
-                onClick={() => {
-                  core.getTool('CropPage').applyCrop();
-                  dispatch(actions.closeElement('annotationPopup'));
-                }}
-              />
-            )}
-            {firstAnnotation.isContentEditPlaceholder() && firstAnnotation.getContentEditType() === window.Core.ContentEdit.Types.TEXT && (
+      ) : shortCutKeysFor3DVisible && firstAnnotation instanceof Annotations.Model3DAnnotation ? (
+        <ShortCutKeysFor3DComponent />
+      ) : (
+        <CustomizablePopup dataElement="annotationPopup">
+          {showCommentButton && (
+            <ActionButton
+              dataElement="annotationCommentButton"
+              title="action.comment"
+              img="circle-right-solid"
```

Comment:
Add comment here to show that this is something we changed. Ideally, do this every time we do some custom changes. It will be much easier to merge with newer versions of PDFtron in the future etc.
```suggestion
              // CUSTOM WISEFLOW: Changed icon
              img="circle-right-solid"
```